rpc/Serve_test.go

```diff
@@ -0,0 +1,47 @@
+package rpc_test
+
+import (
+	"context"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"capnproto.org/go/capnp/v3/rpc"
+)
+
+//clientHook := {}
+
+func TestServe(t *testing.T) {
+	lis, err := net.Listen("tcp", ":8888")
```

Comment:
Let's use `:0` so that this test doesn't fail if something is already bound to `8888`?

Reply:
Oh I had to look that one up. :0 is the next available port. Nifty!